Fibonacci_Number/Fibonacci_Number.cpp

```diff
@@ -0,0 +1,46 @@
+#include<iostream>
+using namespace std;
+
+// Function to print nth fibonacci number
+int FibonacciNumber (int num) 
+{ 
+    if (num <= 1) 
+        return num; 
+    return FibonacciNumber(num - 1) + FibonacciNumber(num - 2); 
+} 
+
+// Function to print fibonacci series  
+int FibonacciSeries (int num)  
+{  
+    int a = 0, b = 1, i;  
+
+    if (num < 1)  
+        return 0;  
+
+    for (i = 1; i <= num; i++)  
+    {  
+        cout<<b<<" ";  
+        int c = a + b;  
+        a = b;  
+        b = c;  
+    }  
+} 
+
+int main() 
+{ 
+    int num; 
+    cin>>num;
+    cout<<num<<"th fibonacci number is "<<FibonacciNumber(num);
```

Comment:
Please find some better print statement,
as *th* won't go with all the numbers.

for 1, it should be 1st and not 1th